README.md

```diff
@@ -70,6 +73,11 @@ CREATE DATABASE chitter_test;
 CREATE TABLE peeps (peep_id SERIAL PRIMARY KEY, created_at TIMESTAMP, message_body VARCHAR(60), user_id VARCHAR(60));
 
 
+### Current Progress
+
+# ![Site screenshot](https://github.com/sabrinalord/chitter-challenge/blob/main/screenshot/chitter_screenshot.png)
+
+
```

Comment:
I saw someone doing a separate MYREADME.md, in which they explained how many classes and methods they used. They also used instructions to run the app at the end. 
I am also learing how to do such MYREADME.md with a clear app map.